addons/scratch-notifier/notifier.js

```diff
@@ -332,7 +332,12 @@ export function notifyNewMessages(messages) {
       parent_title: message.parent_title, // Remixes only
     };
     if (!anyNotified && settings.notification_sound === "addons-ping") {
-      new Audio(chrome.runtime.getURL("./addons/scratch-notifier/ping.mp3")).play();
+      // Note: no audio playback in service worker (Chrome)
+      // https://github.com/ScratchAddons/ScratchAddons/issues/3877
+      if (globalThis.Audio) {
+        // Firefox only
+        new Audio(chrome.runtime.getURL("./addons/scratch-notifier/ping.mp3")).play();
+      }
```

Comment:
(Optional) If you feel like it, we could theoretically use a [offscreen document](https://developer.chrome.com/docs/extensions/reference/api/offscreen) to play our notification sound.

Reply:
Yes, we could